src/test/java/teammates/test/cases/webapi/GetInstructorPrivilegeActionTest.java

```diff
@@ -20,16 +31,329 @@ protected String getRequestMethod() {
+
+        assertTrue(response.isCanViewStudentInSections());
+
+        assertFalse(response.isCanModifySessionCommentsInSections());
+        assertTrue(response.isCanViewSessionInSections());
+        assertTrue(response.isCanSubmitSessionInSections());
+    }
+
+    @Test
+    protected void testExecute_withSectionNameAndInvalidSessionName() {
+        InstructorAttributes instructor1ofCourse1 = dataBundle.instructors.get("instructor1OfCourse1");
+        StudentAttributes student1InCourse1 = dataBundle.students.get("student1InCourse1");
+        loginAsInstructor(instructor1ofCourse1.getGoogleId());
+        ______TS("Request with course id,"
+                + " section name and invalid session name, return section level privilege.");
+
+        String[] invalidSessionParams = {
+                Const.ParamsNames.COURSE_ID, instructor1ofCourse1.getCourseId(),
+                Const.ParamsNames.SECTION_NAME, student1InCourse1.getSection(),
+                Const.ParamsNames.FEEDBACK_SESSION_NAME, "invalid session",
+        };
+
+        GetInstructorPrivilegeAction a = getAction(invalidSessionParams);
+        InstructorPrivilegeData response = (InstructorPrivilegeData) getJsonResult(a).getOutput();
+
+        assertTrue(response.isCanModifyCourse());
+        assertTrue(response.isCanModifyStudent());
+        assertFalse(response.isCanModifyInstructor());
+        assertFalse(response.isCanModifySession());
+
+        assertTrue(response.isCanViewStudentInSections());
+
+        assertTrue(response.isCanModifySessionCommentsInSections());
+        assertFalse(response.isCanViewSessionInSections());
+        assertFalse(response.isCanSubmitSessionInSections());
+    }
+
+    @Test
+    protected void testExecute_withInvalidSectionName() {
+        InstructorAttributes instructor1ofCourse1 = dataBundle.instructors.get("instructor1OfCourse1");
+        loginAsInstructor(instructor1ofCourse1.getGoogleId());
+        ______TS("Request with course id and invalid section name, return general privilege.");
+
+        String[] invalidSectionParams = {
+                Const.ParamsNames.COURSE_ID, instructor1ofCourse1.getCourseId(),
+                Const.ParamsNames.SECTION_NAME, "invalid section",
+        };
+
+        GetInstructorPrivilegeAction a = getAction(invalidSectionParams);
+        InstructorPrivilegeData response = (InstructorPrivilegeData) getJsonResult(a).getOutput();
+
+        assertTrue(response.isCanModifyCourse());
+        assertTrue(response.isCanModifyStudent());
+        assertFalse(response.isCanModifyInstructor());
+        assertFalse(response.isCanModifySession());
+
+        assertFalse(response.isCanViewStudentInSections());
+
+        assertFalse(response.isCanModifySessionCommentsInSections());
+        assertFalse(response.isCanViewSessionInSections());
+        assertFalse(response.isCanSubmitSessionInSections());
+    }
+
+    @Test
+    protected void testExecute_withInvalidSessionName() {
```

Comment:
There is no test for `courseIdWithValidSessionName`?

BTW, you should specific the full detail of a scenario. `withCourseIdAndInvalidSessionName`.